src/components/LocationAutocomplete/LocationAutocomplete.tsx

```
@@ -199,23 +199,22 @@ const LocationAutocomplete: React.FC<Props> = ({
     };
 
     return (
-        <AutosuggestDeprecated
-            getSuggestions={suggestionsList}
+        <Autosuggest
+            suggestions={suggestionsList}
             suggestionToString={suggestionToString}
             isLoading={isLoading}
-            isFocused={isFocused}
-            defaultInputValue={defaultInputValue}
+            defaultValue={defaultInputValue}
             inputPlaceholder={inputPlaceholder}
             showClearButton={singleLocation}
             clearTitle={clearTooltipLabel}
             noSuggestionsPlaceholder={noSuggestionsPlaceholder}
             listRenderer={renderListPoweredByGoogle}
-            saveSelectedValueToInput={singleLocation}
+            // @ts-ignore
```

why do we need to ignore errors here?